doc/quick-start/Dockerfile

```diff
@@ -5,10 +5,12 @@ WORKDIR /build
 COPY . .
 
 RUN set -eux && \
-  apt-get update && apt-get install -y git build-essential autoconf autogen libtool clang && \
+  sed -i s@/deb.debian.org/@/mirrors.aliyun.com/@g /etc/apt/sources.list && \
+  apt-get clean && apt-get update && apt-get install -y git build-essential autoconf autogen libtool clang && \
   git clone --branch v3.21.12 --recurse-submodules https://github.com/protocolbuffers/protobuf && \
   cd protobuf && ./autogen.sh && ./configure && make -j4 && make install && cd .. && ldconfig && \
-  cargo build --release
+  cargo build --release && \
+  cargo build --release --bin validation_lock_client --target x86_64-unknown-linux-gnu
```

Comment:
The above command `cargo build --release` should build `validation_lock_client`

Reply:
Removed